src/main/java/view_post/use_case/ViewPostResponseModel.java

```diff
@@ -44,14 +64,14 @@ public String getPostBody() {
     /**
      * Return the tags associated with this post.
      */
-    public String[] getPostTags() {
+    public String getPostTags() {
         return postTags;
     }
 
     /**
      * Return a list of replies made to this post.
      */
-    public String[] getPostReplies() {
+    public ArrayList<Integer> getPostReplies() {
```

Comment:
Also, why is `postReplies` an ArrayList? If we want to add to it, then we should add a method to add elements to the list rather than calling this getter and then directly calling the add method of ArrayList.